openreview/arr/process/comment_process.py

```diff
@@ -2,8 +2,8 @@ def process(client, edit, invitation):
 
     def get_thread_id(tree, comment_id, forum):
         thread_id = comment_id
-        while tree[thread_id] != forum:
-            thread_id = tree[thread_id]
+        while tree.get(thread_id, forum) != forum:
```

Comment:
you are doing tree.get twice per cycle in the loop? 

what about the following?

```
thread_id = tree.get(thread_id, forum)
where thread_id != forum:
   thread_id = tree.get(thread_id, forum)
```

I would prefer to avoid using "while" because it can lead to infinite loops, I would prefer use a for where we have a start and an end. 



```